docs/config/development/feature-from-scratch/index.rst

```diff
@@ -0,0 +1,385 @@
+
+    This addon will allow us to create distributors within feature config files.
+
+    .. code-block:: yaml
+        :caption: pack.yml
+        :linenos:
+        :emphasize-lines: 7
+
+        id: YOUR_PACK_ID
+
+        version: 0.1.0
+
+        addons:
+          ...
+          config-distributors: "1.+"
+
+    Configure the ``oak_trees.yml`` config to utilize ``PADDED_GRID`` distributor type as shown below.
+
+    .. code-block:: yaml
+        :caption: oak_trees.yml
+        :linenos:
+        :emphasize-lines: 4-8
+
+        id: OAK_TREES
+        type: FEATURE
+
+        distributor:
+          type: PADDED_GRID
+          width: 12
+          padding: 4
+          salt: 5864
+
+    The ``PADDED_GRID`` distributor type utilizes cells in a grid with
+    the feature placed within each cell with padding between each cell
+    to ensure that features don't generate too close to one another.
+
+    ``PADDED_GRID`` utilizes the parameters ``width``, ``padding``, and ``salt``.
+
+    * ``Width`` - Determines the size of each cell that will contain your feature
+    * ``Padding`` - Determines the gap between each cell
+    * ``Salt`` - Offsets the results of the distributor to prevent overlap
+
+5. Create the feature locator
+-----------------------------
+
+.. card::
+
+    Locators determine the y-axis placement of a feature in the world.
+
+    Add the ``config-locators`` addon to the pack manifest, using versions ``1.+``.
+
+    This addon will allow us to create locators within feature config files.
+
+    .. code-block:: yaml
+        :caption: pack.yml
+        :linenos:
+        :emphasize-lines: 7
+
+        id: YOUR_PACK_ID
+
+        version: 0.1.0
+
+        addons:
+          ...
+          config-locators: "1.+"
+
+    Configure the ``oak_trees.yml`` config to utilize the ``TOP`` locator type as shown below.
+
+    .. code-block:: yaml
+        :caption: oak_trees.yml
+        :linenos:
+        :emphasize-lines: 7-11
+
+        id: OAK_TREES
+        type: FEATURE
+
+        distributor:
+          ...
+
+        locator:
+          type: TOP
+          range:
+            min: 0
+            max: 319
+
+    The ``TOP`` locator type will place the feature on the block located at the highest possible y-level.
+
+.. tip::
+
+    You can utilize multiple locators for stricter criteria as shown below with the ``AND`` locator.
+
+    .. code-block:: yaml
+        :caption: feature.yml
+        :linenos:
+        :emphasize-lines: 2
+
+        locator:
+          type: AND
+          locators:
+            - type: TOP
+              range: &range  #range values anchored for other locators to use
+                min: 0
+                max: 319
+            - type: PATTERN
+              range: *range  #references previously anchored range values
+              pattern:
+                type: MATCH_SET
+                blocks:
+                  - minecraft:grass_block
+                  - minecraft:dirt
+                offset: -1
```

Comment:
Include a link to the locator config object with a note that the docs for those locators and the fully list of available locators can be found there - Linking to the docs like this is important where relevant as it provides an 'in' route to the documentation that would otherwise be lost if the user does not explore